frontend/src/libs/components/user-profile-sidebar/user-profile-sidebar.tsx

```diff
@@ -0,0 +1,27 @@
+import { useAppSelector } from '#libs/hooks/hooks.js';
```

Comment:
move it to `pages/user-profile/components/profile-sidebar`

Reply:
done